test/e2e/util.go

```diff
@@ -186,6 +186,10 @@ type RCConfig struct {
 	MaxContainerFailures *int
 }
 
+func LogfPrefix(prefixx string, format string, a ...interface{}) {
```

Comment:
I don't understand the reason behind creating this function. Is it really needed?